### ISSUE TITLE
ImportError: No module named Tensorly

### ISSUE THREAD

**Issue by mullachv** [tensorly/tensorly#75]
Installed as:
```
$ source activate mlp3
$ conda install -c tensorly tensorly

```

```
$ python -V
Python 3.5.2 :: Continuum Analytics, Inc.
```

```
In [1]: import tensorly
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-91217a62ed42> in <module>()
----> 1 import tensorly

ImportError: No module named tensorly

In [2]: 
```

```
$ conda list | grep tensor
tensorflow                0.11.0rc2                py35_0    conda-forge
tensorly                  0.4.2                    py36_0    tensorly
```

I am on a Mac. Any help is much appreciated. Thank you

**Comment by mullachv** [comment]
I fixed this by matching (or exceeding) my envs "python" version (which was 3.5.2) to the "Build" version of tensorly (which is py36_0, as shown in the "conda list" command).
```
$ source activate mlp3
$ conda update python
$ python -V
Python 3.6.6 :: Anaconda, Inc.
$ python -c 'import tensorly'
Using numpy backend.
```

**Comment by leewujung** [comment]
An add-on to this closed issue:
I was trying to install tensorly and had the same issue as the above with python=3.7.
I tried downgraded python to 3.6.6 and it worked.

**Comment by JeanKossaifi** [comment]
Thanks for reporting, it's most likely an issue with the way the conda package is built.

**Comment by JeanKossaifi** [comment]
Should be fixed with tensorly 0.4.3
